spec/services/courses/generate_course_code_service_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe Courses::GenerateCourseCodeService do
```

Comment:
```suggestion
describe Courses::GenerateCourseCodeService do
```

Reply:
Any reason why dropping the top level `RSpec` it seems inconsistent between other tests.

Reply:
Discussed